nomenclature/validation_schemas/meta_schema.yaml

```diff
@@ -0,0 +1,36 @@
+$schema: 'https://json-schema.org/draft/2020-12/schema'

```

Comment:
I would leave this whole thing out. 
It was our initial attempt at making the yaml files more robust. In my opinion though I'd drop all the schema files (in a separate PR) as it usually doesn't add that much information but a lot of complexity in our machinery.